ow_lander/README.md

```diff
@@ -136,7 +136,11 @@ the number of joints or any joint names, this header must be updated as well.
 
 ### Package History
 
-A major overhaul of this package was done after switching from JointPositionController
+* A major overhaul of this package was done after switching from JointPositionController
 to JointTrajectoryController for arm services. Nonetheless, this revised version
 is more comptabile with the file generated by the setup_assistant with few manual
 edits. Refer to PR: [Oceanwater 518 switch to using effort controllers joint trajectory controller](https://github.com/nasa/ow_simulator/pull/40) for complete track of the set of changes.
+
+* An F/T sensor was added to the arm on the distal pitch joint. For more information
```

Comment:
I recommend "force/torque" instead of "F/T" for clarity.

Reply:
addressed